packages/amplify_api_rest/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+## 0.0.1
+
+* TODO: Describe initial release.
```

Comment:
Don't forget to update this.

Reply:
will update in our merged package (RestAPI + GraphQL)